// vite/deps/@solana_wallet-adapter-react.js
import {
  ConnectionContext,
  ConnectionProvider,
  WalletContext,
  WalletNotSelectedError,
  WalletProvider,
  useAnchorWallet,
  useConnection,
  useLocalStorage,
  useWallet
} from "/node_modules/.vite/deps/chunk-WP5ROTR7.js?v=e3f16828";
import "/node_modules/.vite/deps/chunk-HVX2K7ZM.js?v=e3f16828";
import "/node_modules/.vite/deps/chunk-7ADWSKN3.js?v=e3f16828";
import "/node_modules/.vite/deps/chunk-ICS5TY5L.js?v=e3f16828";
import "/node_modules/.vite/deps/chunk-SNZVU36D.js?v=e3f16828";
import "/node_modules/.vite/deps/chunk-NHIXJROC.js?v=e3f16828";
import "/node_modules/.vite/deps/chunk-OOXTULA7.js?v=e3f16828";
import "/node_modules/.vite/deps/chunk-FZHJW7PJ.js?v=e3f16828";
import "/node_modules/.vite/deps/chunk-BUSDCBXK.js?v=e3f16828";
import "/node_modules/.vite/deps/chunk-MVEJMUOB.js?v=e3f16828";
export {
  ConnectionContext,
  ConnectionProvider,
  WalletContext,
  WalletNotSelectedError,
  WalletProvider,
  useAnchorWallet,
  useConnection,
  useLocalStorage,
  useWallet
};
